src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/browser-item-base/browser-item-base.component.theme.scss

```diff
@@ -16,11 +16,31 @@
         background-color: mat-color($primary, surface-mid);
 
       }
+
+      &__basket-icon {
+        mat-icon {
+          background-color: rgba(0, 0, 0, .4);
```

Comment:
Shouldn't variables like the ones in line 16 for example be used instead of "magic" values?